modules/grandpa/src/lib.rs

```diff
@@ -273,16 +284,20 @@ pub mod pallet {
 		}
 	}
 
-	/// The current number of requests which have written to storage.
+	/// Number mandatory headers that we have accepted in the current block for free (returning
```

Comment:
Nit: Isn't this the number of mandatory headers which we are still accepting in this block for free ?